app_dart/test/request_handlers/postsubmit_luci_subscription_test.dart

```diff
@@ -108,23 +120,27 @@ void main() {
       githubChecksUtil: mockGithubChecksUtil,
     );
     scheduler = FakeScheduler(
-      ciYaml: exampleConfig,
       config: config,
       luciBuildService: luciBuildService,
     );
+    ciYamlFetcher = FakeCiYamlFetcher();
     handler = PostsubmitLuciSubscription(
       cache: CacheService(inMemory: true),
       config: config,
       authProvider: FakeAuthenticationProvider(),
       githubChecksService: mockGithubChecksService,
       datastoreProvider: (_) => DatastoreService(config.db, 5),
       scheduler: scheduler,
+      ciYamlFetcher: ciYamlFetcher,
     );
     request = FakeHttpRequest();
-
     tester = SubscriptionTester(request: request);
   });
 
+  tearDown(() {
+    printOnFailure('LOGGER BUFFER:\n${logs.join('\n')}');
```

Comment:
Neat. Why worry about an array and just use a single StringBuffer?

Reply:
I think this goes away with https://github.com/flutter/cocoon/pull/4334 assuming we go in that direction.